### PR TITLE
Extend German translation

### PR DESCRIPTION
**Pull request by nichtich**
I've extended and slightly polished the German user interface and examples.

### CODE REVIEW

File: coursedata/texts/de.yaml [hunk 1]
```diff
@@ -1,21 +1,21 @@
 ui:
-    level_title: "Stufe"
+    level_title: "Level"
```
Comment (Felienne) [inline coursedata/texts/de.yaml:2]: This seems to be quite a big change, is `stufe` not right here? (I don't know a lot of German, so cc @dpprdan)

**Comment by Felienne** [comment]
Nice work on the umlauts!! I think those examples stem from the days before we had unicode variable names!

I am not sure about stufe/level though, it seems a big change to me (but in Dutch we also use the English `level` and not the more Dutch sounding `niveau`, so I totally believe it makes sense!)

**Comment by dpprdan** [comment]
"Level" is better, indeed. It's the same as in Dutch, "Level" is a borrowed word/loanword ("Lehnwort" in German), but is more appropriate in this context, especially given the target group. Also adds to the gamification. 😉 

I just skimmed over the other translations, but they look good to me as well. Nice work @nichtich!

**Comment by Felienne** [comment]
> "Level" is better, indeed. It's the same as in Dutch, "Level" is a borrowed word/loanword ("Lehnwort" in German), but is more appropriate in this context, especially given the target group. Also adds to the gamification. 😉
> 
> I just skimmed over the other translations, but they look good to me as well. Nice work @nichtich!

Cool, thanks for letting us know!!

**Review by Felienne — APPROVED** [review]

**Pull request merged**